README.md

```
@@ -4,103 +4,83 @@ This software is an SMS Bridge Consolidator that works on a laptop connected to
-### 📁 ansible-k3s/
-Kubernetes-based deployment using K3s and `kubernetes.core` Ansible collection
-- Production-ready orchestration
-- Built-in monitoring and scaling
-- See [ansible-k3s/README.md](ansible-k3s/README.md) for details
-
-### 📁 Shared Files
-- `vault.yml` - Encrypted secrets (create from `docs/example_vault.yml`)
-- `schema.sql` - Database schema
-- `sms_server.py` - Main SMS processing application
-- `checks/` - Validation modules
+### 📁 Core Application
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)